images/php-fpm/Dockerfile

```diff
@@ -10,12 +10,12 @@ ENV COMPOSER_HOME=
 ENV MAILHOG_HOST    mailhog
 ENV MAILHOG_PORT    1025
 
-RUN apt update
-
-RUN apt install -y pv sudo bind9-utils python3-pip bash-completion rsync socat oathtool \
-        wget ack automake autoconf nano vim cron mariadb-server
-
-RUN apt install -y python2 && ln -s /usr/bin/python2 /usr/bin/python
+RUN set -eux; \
```

Comment:
What exactly does setting this flag do? I've seen its usage before, but unclear on what exactly it does.

Reply:
https://linuxcommand.org/lc3_man_pages/seth.html
>       -e  Exit immediately if a command exits with a non-zero status.
>       -u  Treat unset variables as an error when substituting.
>       -x  Print commands and their arguments as they are executed.

Reply:
Ah, thank you!